string.go

```diff
@@ -145,11 +149,17 @@ func Words(str string) []string {
 }
 
 // Capitalize converts the first character of string to upper case and the remaining to lower case.
-func Capitalize(word string) string {
-	if len(word) == 0 {
-		return word
+func Capitalize(str string) string {
+	if len(str) == 0 {
+		return str
+	}
+	runes := []rune(str)
+	for i, r := range runes {
+		if i == 0 {
+			runes[i] = unicode.ToUpper(r)
+		} else {
+			runes[i] = unicode.ToLower(r)
```

Comment:
This code still looks strange to me

There was a function now deprecated that did this https://pkg.go.dev/strings#Title

It had been replaced by https://pkg.go.dev/golang.org/x/text/cases#Title

Reply:
Thank you for your suggested changes.